src/index.html

```diff
@@ -21,7 +24,414 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="header__logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptopscomputers"
```

Comment:
The `href` attribute value `#laptopscomputers` is missing a separator between 'laptops' and 'computers'. Consider using a valid character like a hyphen or underscore, e.g., `#laptops-computers` or `#laptops_and_computers`, to ensure proper navigation functionality.